### ISSUE TITLE
multiline input

### ISSUE THREAD

**Issue by glycerine** [go-interpreter/gi#11]
this is a great easy short project to get started with. The `gi` interpreter needs to accept multiple lines rather than just one, when the input is insufficient to complete a statement or expression.

**Comment by glycerine** [comment]
I'm working on this.

**Comment by glycerine** [comment]
This landed with v0.3.1.

Small demo:
~~~
$ gi
====================
gi: a go interpreter
====================
https://github.com/go-interpreter/gi
Copyright (c) 2018, Jason E. Aten, Ph.D.
License: 3-clause BSD. See the LICENSE file at
https://github.com/go-interpreter/gi/blob/master/LICENSE
====================
  [ gi is an interactive Golang environment,
    also known as a REPL or Read-Eval-Print-Loop ]
  [ type ctrl-d to exit ]
  [ type :help for help ]
  [ gi -h for flag help ]
  [ gi -q to start quietly ]
====================
built: '2018-01-08T21:01:52-0600'
last-git-commit-hash: '2a6d0eed127f7eebcd078f7d6d2e18d875e3403d'
nearest-git-tag: 'v0.3.0'
git-branch: 'master'
go-version: 'go_version_go1.9.1_darwin/amd64'
luajit-version: 'LuaJIT_2.1.0-beta3_--_Copyright_(C)_2005-2017_Mike_Pall._http://luajit.org/'
==================
using this prelude directory: '/Users/jaten/go/src/github.com/go-interpreter/gi/pkg/compiler'
using these files as prelude: array.lua, map.lua, prelude.lua, slice.lua, struct.lua
gi> func f() {
>>>    println("hi there" +
>>>    "multiline" +
>>>    "support!!!")
>>>    
>>>    
>>>    }
func f() {

	println("hi there" +

		"multiline" +

		"support!!!")

}
gi> f()
hi theremultilinesupport!!!

gi> [EOF]
$

~~~

**Comment by glycerine** [comment]
@elliott5 @sbinet I'm pretty stoked about landing multiline support. It makes `gi` potentially usable for some real things.
